### PR TITLE
chore: aesthetic updates to tests

### PR DESCRIPTION
**Pull request by KennedyTedesco**
Hi everyone!

I'm excited to make my first contribution to the project. I recently started learning about parsers and interpreters, and I've been really interested in exploring the subject further. As a beginner in both the subject and Rust, working on this project is definitely a great way to learn.

I noticed that there were some areas of the codebase where the style and formatting could be improved, so I took the time to make some changes.

Are you guys open to accepting this kind of change?

Cheers!

### CODE REVIEW

**Comment by azjezz** [comment]
Hey @KennedyTedesco 

Yes, we are open to all kinds of contribution.

However, we prefer that these type of changes are done in one PR, so if you are planning on doing more similar fixes, please appending them to this PR instead of creating a new one, and let me know when it's ready for review.

**Comment by KennedyTedesco** [comment]
Sure, I'll make other tweaks in the test suite. I'll let you know. o/

**Comment by KennedyTedesco** [comment]
Done!

I've made some changes by separating the reading and processing of the test. However, being honest, I'm not sure if this type of refactoring is necessary or desired. My approach may not align with the goals of the project. If you agree, I will focus on bug fixes and new features instead of refactoring things in the future to avoid causing any issues with review processes.

Feel free to close the PR if it brings much noise to the code base. :blush:

**Comment by ryangjchandler** [comment]
Bug fixes and features are preferred, we can focus on tidy up etc once we're "stable".

**Comment by KennedyTedesco** [comment]
I agree with this perspective. ✌️

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 /target
 Cargo.lock
+.idea
```
Comment (azjezz) [inline .gitignore:3]: this should be removed.

**Review by azjezz — CHANGES_REQUESTED** [review]
I think this is okay to merge, only one issue.

**Review by azjezz — APPROVED** [review]

**Pull request merged**